src/measure_theory/group.lean

```diff
@@ -43,73 +55,156 @@ namespace measure
+  ext1 s hs, rw [μ.inv.inv_apply hs, μ.inv_apply, set.inv_inv],
+  exact measurable_inv hs
 end
 
 variables {μ : measure G}
 
-lemma regular.conj [t2_space G] (hμ : μ.regular) : μ.conj.regular :=
+@[to_additive]
+lemma regular.inv [t2_space G] (hμ : μ.regular) : μ.inv.regular :=
 hμ.map (homeomorph.inv G)
 
 end measure
 
-section conj
+section inv
 variables [measurable_space G] [group G] [topological_space G] [topological_group G] [borel_space G]
   {μ : measure G}
 
-@[simp] lemma regular_conj_iff [t2_space G] : μ.conj.regular ↔ μ.regular :=
-by { refine ⟨λ h, _, measure.regular.conj⟩, rw ←μ.conj_conj, exact measure.regular.conj h }
+@[simp, to_additive] lemma regular_inv_iff [t2_space G] : μ.inv.regular ↔ μ.regular :=
+by { refine ⟨λ h, _, measure.regular.inv⟩, rw ←μ.inv_inv, exact measure.regular.inv h }
 
-lemma is_right_invariant_conj' (h : is_left_invariant μ) :
-  is_right_invariant μ.conj :=
+@[to_additive]
+lemma is_mul_right_invariant_inv' (h : is_mul_left_invariant μ) :
+  is_mul_right_invariant μ.inv :=
 begin
-  intros g A hA, rw [μ.conj_apply (measurable_mul_right g hA), μ.conj_apply hA],
+  intros g A hA, rw [μ.inv_apply (measurable_mul_right g hA), μ.inv_apply hA],
   convert h g⁻¹ (measurable_inv hA) using 2,
   simp only [←preimage_comp, ← inv_preimage],
   apply preimage_congr, intro h, simp only [mul_inv_rev, comp_app, inv_inv]
 end
 
-lemma is_left_invariant_conj' (h : is_right_invariant μ) : is_left_invariant μ.conj :=
+@[to_additive]
+lemma is_mul_left_invariant_inv' (h : is_mul_right_invariant μ) : is_mul_left_invariant μ.inv :=
 begin
-  intros g A hA, rw [μ.conj_apply (measurable_mul_left g hA), μ.conj_apply hA],
+  intros g A hA, rw [μ.inv_apply (measurable_mul_left g hA), μ.inv_apply hA],
   convert h g⁻¹ (measurable_inv hA) using 2,
   simp only [←preimage_comp, ← inv_preimage],
   apply preimage_congr, intro h, simp only [mul_inv_rev, comp_app, inv_inv]
 end
 
-@[simp] lemma is_right_invariant_conj : is_right_invariant μ.conj ↔ is_left_invariant μ :=
-by { refine ⟨λ h, _, is_right_invariant_conj'⟩, rw ←μ.conj_conj, exact is_left_invariant_conj' h }
+@[simp, to_additive]
+lemma is_mul_right_invariant_inv : is_mul_right_invariant μ.inv ↔ is_mul_left_invariant μ :=
+by { refine ⟨λ h, _, is_mul_right_invariant_inv'⟩, rw ← μ.inv_inv,
+     exact is_mul_left_invariant_inv' h }
+
+@[simp, to_additive]
+lemma is_mul_left_invariant_inv : is_mul_left_invariant μ.inv ↔ is_mul_right_invariant μ :=
+by { refine ⟨λ h, _, is_mul_left_invariant_inv'⟩, rw ← μ.inv_inv,
+     exact is_mul_right_invariant_inv' h }
+
+end inv
+
+namespace measure
+
+variables [measurable_space G] [topological_space G] [borel_space G] {μ : measure G}
+
+section group
+
+variables [group G] [topological_group G]
+
+/-! Properties of regular left invariant measures -/
+
+@[to_additive]
+lemma null_of_is_mul_left_invariant_of_ne_zero (hμ : μ.regular) (h2μ : is_mul_left_invariant μ)
```

Comment:
The lemma does not convey the fact that you are proving an iff statement. `is_mul_left_invariant.null_iff_empty`? (I'd leave the `of_ne_zero` as it is an obvious and minor side condition). And the next one could be `is_mul_left_invariant.null_iff`. Same thing for the other ones. I feel that the dot notation would make it is easier to read and use.

Reply:
For the dot notation I wasn't sure whether to put it in the `is_mul_left_invariant` or in the `regular` namespace. I agree the name name is not good.